base/operators.jl

```diff
@@ -1300,14 +1300,10 @@ contains `missing` but not `item`, in which case `missing` is returned
 matching the behavior of [`any`](@ref) and [`==`](@ref)).
 
 Some collections follow a slightly different definition. For example,
-[`Set`](@ref)s check whether the item [`isequal`](@ref) to one of the elements.
-[`Dict`](@ref)s look for `key=>value` pairs, and the key is compared using
-[`isequal`](@ref). To test for the presence of a key in a dictionary,
-use [`haskey`](@ref) or `k in keys(dict)`. For these collections, the result
-is always a `Bool` and never `missing`.
-
-To determine whether an item is not in a given collection, see [`∉`](@ref).
-You may also negate the `in` by doing `!(a in b)` which is logically similar to "not in".
+[`Set`](@ref)s check whether the item [`isequal`](@ref) to one of the elements;
+[`Dict`](@ref)s look for `key=>value` pairs, and the `key` is compared using
+[`isequal`](@ref). To test for the presence of a key in a dictionary, use [`haskey`](@ref)
+or `k in keys(dict)`. For these collections, the result is always a `Bool` and never `missing`.
```

Comment:
Last sentence is literally contradicted by the `(1=>missing) in Dict(1=>10, 2=>20)` example.

Reply:
I don't think so. The last sentence refers to the sentence before it: "To test for the presence of a key in a dictionary, use [`haskey`](@ref) or `k in keys(dict)`. " (which does always return a `Bool`). But maybe this can be split out into a separate paragraph.

Reply:
Then it can make sense. But as written it sounds like it applies to the whole Set / Dict part of the paragraph, not the "haskey" sentence. That could be "For these tests", not "these collections".

But what you write with parentheses looks better, to make this part of that sentence.